crates/oxc_ast/src/generated/derive_clone_in.rs

```diff
@@ -163,6 +317,14 @@ impl<'new_alloc> CloneIn<'new_alloc> for IdentifierReference<'_> {
             reference_id: Default::default(),
         }
     }
+
+    fn clone_in_with_semantic_ids(&self, allocator: &'new_alloc Allocator) -> Self::Cloned {
+        IdentifierReference {
+            span: CloneIn::clone_in_with_semantic_ids(&self.span, allocator),
+            name: CloneIn::clone_in_with_semantic_ids(&self.name, allocator),
+            reference_id: Default::default(),
+        }
+    }
```

Comment:
This PR doesn't achieve what it's trying to.

`clone_in_with_semantic_ids` does exactly the same as `clone_in`. It doesn't maintain the semantic IDs!

Reply:
Yeah, the original pr is pretty stale, some parts of the code are not synced